client/src/components/NumberField/tests/NumberField-story.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { storiesOf, setAddon } from '@storybook/react';
+import actionListMaker from '../../../stories/actionListMaker';
+import { withKnobs, text, boolean } from '@storybook/addon-knobs/react';
+import JSXAddon from 'storybook-addon-jsx';
+import NumberField from 'components/NumberField/NumberField';
+
+setAddon(JSXAddon);
+
+const tagActions = actionListMaker('onClick', 'onChange');
+
+storiesOf('Admin/NumberField', module)
+  .addDecorator(story => (
+    <div>
+      {story()}
+    </div>
+  ))
+  .addDecorator(withKnobs)
+  .addWithJSX('NumberField', () => (
+    <NumberField
+      title={text('title', '')}
```

Comment:
Maybe add some sensible default example values to your knobs.

All form fields support a `message` props as well to display validation errors.
```js
    message={{
        type: select('message type', ['', 'error'], 'error'),
        value: text('message', 'A message about the field.')
      }}
```